hardware-and-software-requirements.md

```diff
@@ -55,6 +55,7 @@ TiDB 支持部署和运行在 Intel x86-64 架构的 64 位通用硬件服务器
 | PD | 4 核+ | 8 GB+ | SAS, 200 GB+ | 千兆网卡 | 1（可与 TiDB 同机器） |
 | TiKV | 8 核+ | 32 GB+ | SSD, 200 GB+ | 千兆网卡 | 3 |
 | TiFlash | 32 核+ | 64 GB+ | SSD, 200 GB+ | 千兆网卡 | 1 |
+| TiCDC | 8 核+ | 16 GB+ | SSD, 200 GB+ | 千兆网卡 | 1 |
```

Comment:
Is `SAS` enough here?

Reply:
It should be enough if we not enable file sort.
```suggestion
| TiCDC | 8 核+ | 16 GB+ | SAS, 200 GB+ | 千兆网卡 | 1 |
```